trustmanager/x509utils.go

```diff
@@ -440,7 +477,12 @@ func EncryptPrivateKey(key data.PrivateKey, passphrase string) ([]byte, error) {
 		return nil, err
 	}
 
-	return pem.EncodeToMemory(encryptedPEMBlock), nil
+	if alias != "" && encryptedPEMBlock.Headers != nil {
```

Comment:
Are the headers ever nil?  Don't the headers for an encrypted PEM block have to include the algorithm (e.g. should there be an error if the headers are nil?)